stm32cube/stm32wb0x/drivers/src/stm32wb0x_hal_radio.c

```diff
@@ -204,15 +204,7 @@ static uint8_t CondRoutineRxTrue(ActionPacket *p)
 /* Round up HOT_TABLE_SIZE to an integer number of words then add 4 words for table management pointers */
 volatile uint32_t hot_table_radio_config_u32[((HOT_TABLE_SIZE + 3) >> 2) + 4] = {0x00};
 
-/* BLUE RAM, reserved for radio communication. Not usable from the application */
-__SECTION(".bss.__blue_RAM")
-#if defined(STM32WB05) || defined(STM32WB09)
-__REQUIRED(uint8_t __blue_RAM[CFG_NUM_RADIO_TASKS * 92 + 28]) = {0,};
-#elif defined(STM32WB06) || defined (STM32WB07)
-__REQUIRED(uint8_t __blue_RAM[CFG_NUM_RADIO_TASKS * 80 + 28]) = {0,};
-#else
-#warning "No Blue RAM allocated"
-#endif
```

Comment:
For someone which may want to do further modification in those files, it maybe easier to have this defined under `#if 0` so that symbols are still there if one searches them.

Reply:
Same for other patches